src/main/scala/it/pagopa/interop/purposeprocess/api/impl/PurposeApiServiceImpl.scala

```diff
@@ -431,6 +432,62 @@ final case class PurposeApiServiceImpl(
     }
   }
 
+  override def clonePurpose(purposeId: String)(implicit
+    contexts: Seq[(String, String)],
+    toEntityMarshallerPurpose: ToEntityMarshaller[Purpose],
+    toEntityMarshallerProblem: ToEntityMarshaller[Problem]
+  ): Route =
+    authorize(ADMIN_ROLE, API_ROLE, SECURITY_ROLE) {
```

Comment:
Write operations on purposes can be done only by admins
```suggestion
    authorize(ADMIN_ROLE) {
```